lib/notification-issue.coffee

```diff
@@ -70,7 +70,7 @@ class NotificationIssue
 
     if title.length > TITLE_CHAR_LIMIT
       title = title.substring(0, TITLE_CHAR_LIMIT - 3) + '...'
-    title
+    title.replace(/\r?\n|\r/g,"");
```

Comment:
@ungb: When you have a moment, can you take a look at the [build failure related to this change](https://travis-ci.org/atom/notifications/builds/234162685?utm_source=github_status&utm_medium=notification)?